packages/connect-explorer/package.json

```diff
@@ -34,7 +34,8 @@
         "redux-thunk": "^2.3.0",
         "stringify-object": "^3.3.0",
         "styled-components": "^5.3.3",
-        "trezor-connect": "8.2.7-beta.4"
+        "trezor-connect": "8.2.7-beta.4",
+        "@trezor/components": "*"
```

Comment:
alphabetical order, @trezor should be at the top

Reply:
fixed